src/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
    image: "mojaloop/sdk-scheme-adapter:latest"
    env_file: ./scheme-adapter.env
    ports:
-     - "3500:3000"
-     - "3501:4000"
+     - "3500:4000"
+     - "3501:4001"
```

Comment:
Please change this to `4001:4001`

Reply:
see my above comment about docker for mac compatibility